human-state-provider/conf/human-state-provider.ini

```diff
@@ -1,4 +1,4 @@
 playback true
 segments (("xsens_world", "Pelvis"),("xsens_world", "L5"),("xsens_world", "L3"),("xsens_world", "T12"),("xsens_world", "T8"),("xsens_world", "Neck"),("xsens_world", "Head"),("xsens_world", "RightShoulder"),("xsens_world", "RightUpperArm"),("xsens_world", "RightForeArm"),("xsens_world", "RightHand"),("xsens_world", "LeftShoulder"),("xsens_world", "LeftUpperArm"),("xsens_world", "LeftForeArm"),("xsens_world", "LeftHand"),("xsens_world", "RightUpperLeg"),("xsens_world", "RightLowerLeg"),("xsens_world", "RightFoot"),("xsens_world", "RightToe"),("xsens_world", "LeftUpperLeg"),("xsens_world", "LeftLowerLeg"),("xsens_world", "LeftFoot"),("xsens_world", "LeftToe"))
-human_urdf XSensURDF_subj1.urdf
+human_urdf /Users/makaveli/Downloads/XSensURDF_subj1.urdf
```

Comment:
This suggests me two things:
 * you changed your machine but you didn't change your hostname (which is bad because it's like renaming your dog with the same name of your past dog)
 * you use the Download folder as your download folder (which is another bad thing)